modules/fox-spring/src/main/java/com/ensolvers/fox/spring/lightweightcontainer/GlobalApplicationContext.java

```diff
@@ -0,0 +1,122 @@
+package com.ensolvers.fox.spring.lightweightcontainer;
+
+import com.ensolvers.fox.services.logging.Logger;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.lang.reflect.Method;
+import java.util.Collection;
+import java.util.List;
+import java.util.Vector;
+import org.springframework.beans.BeansException;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.ApplicationContextAware;
+import org.springframework.context.ApplicationContextException;
+import org.springframework.context.support.ClassPathXmlApplicationContext;
+import org.springframework.core.io.Resource;
+import org.springframework.core.io.support.PathMatchingResourcePatternResolver;
+import org.springframework.stereotype.Component;
+
+
+/**
+ *      GlobalApplicationContext is a Spring companion.   <p>
+ *      Has two main features: look up and build.   <p>
+ *      <p>
+ *      <p>
+ *      * Look up allows to get from Spring context any registered bean  <p>
+ *      Example:  <code> MyComponent component = (MyComponent)GlobalApplicationContext.getBean( "MyComponent" );</code> <p>
+ *      <p>
+ *      <p>
+ *      * Construction allows building on the fly new context by offering legacy configuration xml files. <p>
+ *      Example:  <code> MyComponent component = (MyComponent)GlobalApplicationContext.getBean( "MyComponent" );</code> <p>
+ *
+ *      <p>
+ *
+ */
+@Component
+public class GlobalApplicationContext implements ApplicationContextAware {
```

Comment:
Add documentation to all public methods following general standards of Java open source projects

Reply:
fixed